src/components/settings/SettingsOption.tsx

```diff
@@ -122,18 +24,27 @@ export const SettingsOption = forwardRef(function RowLayout(
         disabled,
         onClick,
         iconClassName,
+        className,
         ...rest
     }: SettingsOptionProps,
     forwardedRef: React.Ref<HTMLDivElement>,
 ) {
-    const { isDark, getThemeColor } = useThemeMode();
-
     return (
-        <StyledRow
-            variant={variant}
-            isDark={isDark()}
+        <div
+            className={cn(
+                'relative flex w-full max-h-13 p-4 cursor-pointer gap-3 items-center border-none bg-none focus-visible:-outline-offset-2',
+                {
+                    'transition-smoothEase': !isFirefox,
+                    'transition-firefoxSmoothEase focus-visible:outline-2 focus-visible:-outline-offset-2':
+                        isFirefox,
+                    'hover:bg-theme-secondary-50 dark:hover:bg-theme-secondary-700':
+                        variant === 'primary',
+                    'hover:bg-theme-error-50 dark:hover:bg-[rgba(204,28,0,0.10)] text-theme-error-600 dark:text-theme-error-500 group':
```

Comment:
```suggestion
                    'hover:bg-theme-error-50 dark:hover:bg-theme-error-800/20 text-theme-error-600 dark:text-theme-error-500 group':
```

it's better if you compare against the design to see if they use a theme color with an opacity applied, and do the same in tailwind with the opacity modifiers